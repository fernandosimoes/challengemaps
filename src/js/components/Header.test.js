import React from 'react';
import Header from './Header';
import renderer from 'react-test-renderer';

test('Link Header Render', () => {

    let component = renderer.create(
        <Header />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

