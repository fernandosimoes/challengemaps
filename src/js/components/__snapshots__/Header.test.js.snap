// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link Header Render 1`] = `
<div
  className="header"
>
  <div
    className="container"
  >
    <h1>
      Desempenho das lojas
    </h1>
  </div>
</div>
`;
